src/io.jl

```diff
@@ -180,7 +181,7 @@ function _load(png_ptr, info_ptr; gamma::Union{Nothing,Float64}=nothing, expand_
 
     @debug(
         "Read PNG info:",
-        fpath,
+        png_ptr,
```

Comment:
The file pointer serves as the primary key to connect two pieces of logging information.